lib/ansible/galaxy/api.py

```diff
@@ -696,17 +697,29 @@ def wait_import_task(self, task_id, timeout=0):
         start = time.time()
         wait = 2
 
+        token_expired = False
         while timeout == 0 or (time.time() - start) < timeout:
+            if isinstance(self.token, KeycloakToken) and self.token.expired:
```

Comment:
`isinstance()` is relatively expensive. How about moving the detection outside the loop and just reusing the result here?
```python
is_keycloak_token = isinstance(self.token, KeycloakToken)
```

```suggestion
            if is_keycloak_token and self.token.expired:
```